tests/test_forcedPhot.py

```diff
@@ -79,6 +146,100 @@ def testRun(self):
         self.assertFloatsNotEqual(measCat["base_TransformedCentroid_x"], self.refCat['truth_x'])
         self.assertFloatsNotEqual(measCat["base_TransformedCentroid_y"], self.refCat['truth_y'])
 
+    def testRunQuantum(self):
```

Comment:
Is this a good pattern to follow for pipeline tasks in general? I might be tempted to cargo cult it.